x-pack/plugins/discover_log_explorer/README.md

```diff
@@ -0,0 +1,3 @@
+# Discover Log Explorer
+
+This plugin exposes and registers Logs+ features.
```

Comment:
Do we want to add a more accurate description here (and in the Kibana manifest)? Something that explains why the plugin name is prefixed with `discover` like:

This plugin registers a `log_explorer` profile with Discover, which offers several affordances specific for log consumption:

1. first feature
2. second feature
3. ...

What do you think?

Reply:
100% agree, I drafted simple sentences and there is a huuuuge lack of documentation right now, I wanted to moe fast on the implementation but I'm ok with adding more descriptive reasons behind this new plugin implementation :)